File: modul6/Python/PRAK605-2310817210023-RifkyPutraMahardika.py
ordo = int(input("Ordo matriks nxn\n"))
matriksA = [[0] * ordo for i in range(ordo)]
matriksB = [[0] * ordo for i in range(ordo)]
hasil = [[0] * ordo for i in range(ordo)]

print("Matriks A")
for i in range(ordo):
    matriksA[i] = list(map(int, input().split()))

print("Matriks B")
for i in range(ordo):
    matriksB[i] = list(map(int, input().split()))

for i in range(ordo):
    for j in range(ordo):
        hasil[i][j] = 0
        for k in range(ordo):
            hasil[i][j] += matriksA[i][k] * matriksB[k][j]

print("Matriks AXB")
for i in range(ordo):
    for j in range(ordo):
        print(hasil[i][j], end=" ")
    print()

File: modul6/C/PRAK605-2310817210023-RifkyPutraMahardika.c
#include<stdio.h>

int main(){
    int i, j, k, ordo;
    int hasil[50][50];
    printf("Ordo matriks nxn\n");
    scanf("%d", &ordo);
    int matriksA[ordo][ordo],matriksB[ordo][ordo];

    printf("Matriks A\n");
    for(i=0;i<ordo;i++){
        for(j=0;j<ordo;j++){
            scanf("%d", &matriksA[i][j]);
        }
    }

    printf("Matriks B\n");
    for(i=0;i<ordo;i++){
        for(j=0;j<ordo;j++){
            scanf("%d", &matriksB[i][j]);
        }
    }

    for(i=0; i<ordo;i++){
        for(j=0; j<ordo; j++) {
            hasil[i][j] = 0;
            for(k=0; k<ordo; k++){
                hasil[i][j] += matriksA[i][k] * matriksB[k][j];
            }
        }
    }

    printf("Matriks AXB\n");
    for(i=0; i<ordo;i++){
        for(j=0; j<ordo; j++) {
            printf("%d ", hasil[i][j]);
        }
        printf("\n");
    }
return 0;
}